agents/plugins/smart

```diff
@@ -108,6 +108,20 @@ if which smartctl > /dev/null 2>&1 ; then
             continue
         fi
 
+        # Exclude removable media
+        if which udevadm > /dev/null 2>&1; then
```

Comment:
We have made the experience that the following function is more robust:
```
# Function to replace "if type [somecmd]" idiom
# 'command -v' tends to be more robust vs 'which' and 'type' based tests
inpath() {
    command -v "${1:?No command to test}" >/dev/null 2>&1
}
```
Simply add this function at the top level and use it here.